backend/client.go

```diff
@@ -139,10 +139,9 @@ func (c *backendClient) TerminateOrchestration(ctx context.Context, id api.Insta
 			return fmt.Errorf("failed to configure termination request: %w", err)
 		}
 	}
-
 	e := helpers.NewExecutionTerminatedEvent(req.Output, req.Recursive)
 	if err := c.be.AddNewOrchestrationEvent(ctx, id, e); err != nil {
-		return fmt.Errorf("failed to add terminate event: %w", err)
+		return fmt.Errorf("failed to terminate orchestration: %w", err)
```

Comment:
I think this error message is a bit misleading. We don't actually terminate the orchestration here. Rather, we're scheduling the termination.

Reply:
Done